terraformInstall.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+## CONDITIONAL 1
+echo "Installing prerequisites..."
+sudo apt-get update && sudo apt-get install -y gnupg software-properties-common
+
+#!/bin/bash
+
+# Check if gpg is installed
+if ( which gpg > /dev/null 2>&1 )
+then
+    echo "gpg is installed."
+else
+    echo "Error: gpg is not installed. Please install it before running this script."
+    exit 1
```

Comment:
Never use exit in any script, the script needs to run to the end. Not sure where yall keep picking this up.

Reply:
I think I'm the only one using the exit, my thought was that if gpg is not installed, the remaining script will not run correctly, so it should stop until the user installs gpg. Actually now that I just typed that out I realize that the user will not know when and where to restart it- I'm an idiot. 
Also- this echo statement is lazy because I don't know what to do for the gpg installation commands- I mean I do know but do I write something for each OS? One bash command? As it is I leave it to the user to look it up and complete the step on their own...